exercises/concept/bird-count/.docs/instructions.md

```diff
@@ -0,0 +1,71 @@
+# Instructions
+
+You're an avid bird watcher that keeps track of how many birds have visited your garden on any given day.
+
+You decided to bring your bird watching to a new level and implement a few tools that will help you track and process the data.
+
+You have chosen to store the data as a list of integers.
+The first number in the list is the number of birds that visited your garden today, the second yesterday, and so on.
+
+```exercism/note
```

Comment:
This is a shame, is there not an example of using recursion where it would be the idiomatic thing to do?

Reply:
Yes, I agree that it's a shame, however I generally think that it's a good exercise to re-implement core functions to deepen your understanding of them. With the understanding that it's not idiomatic of course, and in that sense, a concept exercise is the right place to do it.

Pretty much all practice exercise that are marked with `recursion` are examples where recursion is the idiomatic thing to do.

I do realize that I'm biased because I don't really want to come up with a completely new exercise, but I will:
- tweak the note to emphasize that this is not idiomatic, but for the sake of learning
- come up with an exercise for `tail-call-recursion` that uses recursion idiomatically